src/snowflake/snowpark/dataframe.py

```diff
@@ -1110,6 +1110,79 @@ def to_pandas_batches(
             **kwargs,
         )
 
+    @experimental(version="1.27.0")
+    def to_arrow(
+        self,
+        *,
+        statement_params: Optional[Dict[str, str]] = None,
+        block: bool = True,
+        _emit_ast: bool = True,
+        **kwargs: Dict[str, Any],
+    ):
+        """
+        Executes the query representing this DataFrame and returns the result as a
```

Comment:
can we add a clarification somewhere (maybe not here), saying users have to pip install "snowflake-snowpark-python[pandas]" to use it?